Update/ura_012.txt

```diff
@@ -3467,7 +3467,7 @@ void main()
 			NULL, " ...That's what I'd like to say, but let me offer you a parting gift.", Line_WaitForInput);
 	ModPlayVoiceLS(4, 32, "ps3/s14/32/vt4bb_hanada1101", 256, TRUE);
 	OutputLine(NULL, "誰に殺されるかわからんまま死んで、化けて出られるのも勘弁ですからね。",
-			NULL, " After all, it would be such a pity to die without ever knowing who it was that killed you. ", GetGlobalFlag(GLinemodeSp));
+			NULL, " After all, it would be such a pain if you came back as a ghost after dying without ever knowing who it was that killed you. ", GetGlobalFlag(GLinemodeSp));
```

Comment:
```suggestion
			NULL, " After all, it would be such a pain to come back as a ghost without ever knowing who it was that killed you. ", GetGlobalFlag(GLinemodeSp));
```